lib/rules/define-macros-order.js

```diff
@@ -109,6 +111,12 @@ function create(context) {
   /** @type {ASTNode} */
   let defineExposeNode
 
+  if (order.includes(MACROS_EXPOSE) && defineExposeLast) {
+    throw new Error(
+      'defineExposeLast cannot be used if defineExpose macro is in order array.'
```

Comment:
I'd rather reverse the error message:

```suggestion
      "`defineExpose` macro can't be in the `order` array if `defineExposeLast` is true."
```